platforms/odroid/boot.txt

```diff
@@ -0,0 +1,20 @@
+# After modifying, run ./mkscr
+
+# MAC address configuration
+# setenv macaddr "00:1e:ee:61:7a:39"
+
+# Remove delay before bootargs
+setenv bootdelay 0
+
+part uuid ${devtype} ${devnum}:${bootpart} uuid
+setenv bootargs "console=ttySAC1 console=${console} root=/dev/mmcblk0p2 init=/usr/lib/systemd/systemd rw rootwait smsc95xx.macaddr=${macaddr} quiet"
```

Comment:
`init` and `quiet` were added here

Reply:
How come `init` needs to be explicitly set? `/sbin/init` should symlink to systemd?

I don't mind if there is no reason, just curious if there is!